news/_posts/core-weekly/2020-10-26-coreweekly-43-2020.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 43 of 2020"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2020-10-26
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 19th to Sunday 25th of October 2020.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
```

Comment:
Dangling phrase?

Reply:
I hesitate between remove it or keep it. If I remove it, I have no more "hello ..." statement so the "general messages" feel a bit cold, informative 😅 

Hard to find the right tone